corporate/tests/test_stripe.py

```diff
@@ -1864,6 +1869,18 @@ def test_update_billing_method_of_current_plan(self) -> None:
         plan.refresh_from_db()
         self.assertEqual(plan.charge_automatically, False)
 
+    @mock_stripe()
+    def test_customer_has_credit_card_as_default_source(self, *mocks: Mock) -> None:
+        iago = self.example_user("iago")
+        customer = Customer.objects.create(realm=iago.realm)
+        self.assertFalse(customer_has_credit_card_as_default_source(customer))
+
+        customer = do_create_stripe_customer(iago)
+        self.assertFalse(customer_has_credit_card_as_default_source(customer))
+
+        customer = do_create_stripe_customer(iago, stripe_token=stripe_create_token().id)
```

Comment:
It's not really apparent why the usage`stripe_token` causes the customer to be with credit card as the default source, should we add a comment to clear that up?

Reply:
@mateuszmandera What part is not clear? The part where stripe_create_token generates a token that has a credit card associated with it or the part where the credit card is made as the default source?